### ISSUE TITLE
Accessibly convey active container to screen reader users

### ISSUE THREAD

**Issue by ndarilek** [mozilla/multi-account-containers#1417]
I'm a screen reader user, and it isn't at all obvious to me whether or not I'm in a container tab. Since I'm now trying to use container tabs more heavily, this lack is significantly hindering their use for me.

I'm interested in doing some extension development, so may be willing to submit a PR. But I need some guidance on where to look. Specifically:

1. How is the user's presence in a container tab currently conveyed? I'm guessing color somehow? Pointers to the specific area in the code and the APIs used would be helpful.

2. Thoughts on how to convey this accessibly are welcome. The only thing I can think of is changing the browser window title to append the container name, doing something similar with the page title, etc. Not sure if browser extensions can do that, if there is a better suggestion, etc.

Thanks.

**Comment by kendallcorner** [comment]
Hi @ndarilek. Yes, containers are conveyed to the user by a color underline of the tab and also by an icon and name in the same  color on the right side of the url. All of that code is part of Firefox itself, but the extension does tell Firefox the name, color, and icon of the containers. I'm not at all familiar with screen reading, so I'm not sure whether a notification of some kind would require extension work or platform (firefox) work.
